### PR TITLE
Update features.md

### PR DESCRIPTION
**Pull request by urrsk**
Update the feature list

### CODE REVIEW

**Comment by urrsk** [comment]
Is the context of this list correct?

**Review by livanov93 — COMMENTED** [review]
@urrsk Only small comment about velocity mode.

File: ur_robot_driver/doc/features.md [hunk 1]
```diff
@@ -2,23 +2,25 @@
...
 | use the driver without a teach pendant necessary      | yes                       |
-| support of CB2 robots                                 | -                         |
+| support of CB2 robots                                 | no                        |
 | trajectory extrapolation on robot on missing packages | yes                       |
 | use ROS as drop-in for TP-programs                    | yes                       |
-| headless mode                                         | not yet implemented       |
-| extract calibration from robot                        | not yet implemented       |
+| headless mode                                         | yes                       |
+| extract calibration from robot                        | yes                       |
 | send custom script commands to robot                  | yes                       |
```
Comment (urrsk) [inline ur_robot_driver/doc/features.md:24]: @livanov93 What about sending script command to the robot. Is that supported in the current version?

File: ur_robot_driver/doc/features.md [hunk 2]
```diff
@@ -2,23 +2,25 @@
...
 | Feature                                               | ROS2 Driver
 | ---                                                   | ---                       |
-| position-based control                                | yes                       |
-| scaled position-based control                         | yes                       |
-| velocity-based control                                | no<sup>1</sup>            |
-| reporting of tcp wrench                               | not yet implemented       |
-| reporting of tcp wrench in tcp link                   | not yet implemented       |
+| joint-position-based control                          | yes                       |
+| scaled joint-position-based control                   | yes                       |
+| joint-velocity-based control                          | no<sup>1</sup>            |
```
[the remark this answers is not in the document]
Comment (urrsk) [reply ur_robot_driver/doc/features.md:7]: Thanks

File: ur_robot_driver/doc/features.md (hunk 1 above)
Comment (livanov93) [reply ur_robot_driver/doc/features.md:24]: @urrsk It is not, I missed this. We don't have a string controller yet, so we only support string commands that are predefined (e.g. payload setting)

Comment (urrsk) [reply ur_robot_driver/doc/features.md:24]: Okay

Comment (urrsk) [reply ur_robot_driver/doc/features.md]: I have made an issue for the missing support of 'send custom script commands to robot' #251

**Comment by urrsk** [comment]
#99

**Review by livanov93 — APPROVED** [review]
It looks good. Thanks a lot!

**Pull request merged**